### DISCUSSION TITLE
When is Linux support expected?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [twinbasic/twinbasic#1616] [Q&A] [answered]
Per [`twinbasic.com/preorder.html`](https://twinbasic.com/preorder.html#:~:text=support%20(planned%20for-,linux,-%2C%20Mac%2C%20Android)), it's purportedly expected. I'm really glad to see this, since I've been unable to touch my VB6 project files since I transferred from Windows to OpenSUSE. However, I don't see an issue at [`github.com/twinbasic/twinbasic/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+Linux`](https://github.com/twinbasic/twinbasic/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+Linux) for me to track closure of.

**Comment by fafalone** [comment]
The most up to date estimates for it are on [the roadmap](https://github.com/twinbasic/twinbasic/issues/335).

**Reply by roke-julian-lockhart** [reply, marked as answer]
https://github.com/twinbasic/twinbasic/discussions/1617#discussioncomment-6452507
